app/assets/stylesheets/juntos_bootstrap/containers/_dashboards.scss

```diff
@@ -20,6 +16,12 @@
       opacity: 1;
       text-decoration: none;
     }
+  }
+
+  &.user-authorization-documents {
+    background-color: $border-gray;
+    border: .3px solid $border-gray;
```

Comment:
are you sure `.3px` border is working? I thought it will only be visible if >1px 🤔